### PR TITLE
Add Daffodil Software, Databiz Software, Grameen Solutions, Metatude and IPvision

### PR DESCRIPTION
**Pull request by mmahmoodictbd**
Add Daffodil Software Limited, Databiz Software Limited, Grameen Solutions Ltd, Metatude Asia Ltd, and IPvision.

### CODE REVIEW

**Comment by mmahmoodictbd** [comment]
@arafat-hasan If you review this, I can merge.

**Review by arafat-hasan — APPROVED** [review]
All newly added companies addresses and websites checked.

Edit: I have found some problem with the addresses.

File: README.adoc [hunk 1]
```diff
@@ -151,6 +156,21 @@ https://github.com/ChumbokIT[Github]
 |Java, Spring Boot, C#, .Net, Angular/Vue/JQuery
 |http://datasoft-bd.com/[Website]
 
+|Daffodil Software Limited (DSL)
+|DF Tower (Level: 11-A), House # 11, Road # 14, Dhanmondi, Dhaka-1209
+|Java, .NET, PHP, Android, iOS
+|http://daffodilsoft.com/[Website]
+
+|Databiz Software Limited
+|Mirpur DOHS Avenue #3. Road #12, House #860-861, Dhaka 1216
```
Comment (arafat-hasan) [inline README.adoc:165]: Hashtag in address should be removed, github renders #number as a pull request link.

File: README.adoc [hunk 2]
```diff
@@ -379,6 +409,11 @@ https://www.linkedin.com/company/magnito-digital/[LinkedIn]
 |Java, Spring
 |http://www.metafour.com[Website]
 
+|Metatude Asia Ltd
+|Mark Mansion (Level 4 & 5), 36 Sonargaon Janapath, Sector #9, Uttara, Dhaka – 1230
```
Comment (arafat-hasan) [inline README.adoc:413]: Hashtag in address should be removed, as GitHub renders #number as a pull request link.

File: README.adoc [hunk 3]
```diff
@@ -151,6 +156,21 @@ https://github.com/ChumbokIT[Github]
 |Java, Spring Boot, C#, .Net, Angular/Vue/JQuery
 |http://datasoft-bd.com/[Website]
 
+|Daffodil Software Limited (DSL)
+|DF Tower (Level: 11-A), House # 11, Road # 14, Dhanmondi, Dhaka-1209
```
Comment (arafat-hasan) [inline README.adoc:160]: Though there is a space after hashtag in the address line, and it's working fine now. But as GitHub renders #number as a pull request link, this hashtag should be removed.

File: README.adoc [hunk 4]
```diff
@@ -188,12 +208,12 @@ https://github.com/ChumbokIT[Github]
 
 |Evaly Limited
 |House #8, Road # 14, Dhanmondi, Dhaka-1209
```
Comment (arafat-hasan) [inline README.adoc:210]: Hashtag in address should be removed, as GitHub renders #number as a pull request link.

File: README.adoc [hunk 5]
```diff
@@ -287,7 +312,7 @@ https://github.com/ChumbokIT[Github]
 
 |Integrated Software and Technologies Ltd.
 |Avenue # 9, Road #9, House #1043 (2nd Floor) DOHS Mirpur, Dhaka, Bangladesh
```
Comment (arafat-hasan) [inline README.adoc:314]: Hashtag in address should be removed, as GitHub renders #number as a pull request link.

File: README.adoc [hunk 6]
```diff
@@ -638,6 +673,11 @@ https://www.linkedin.com/company/southtechgroup/[LinkedIn]
 |Java, PHP, AngularJS
 |https://www.vantage.com/[Website],  https://www.facebook.com/VantageLabsDhaka/[Facebook]
 
+|Vivasoft
+|Floor #16, Ahmed Tower, 28, 30, Kemal Ataturk Ave, Banani, Dhaka 1213
```
Comment (arafat-hasan) [inline README.adoc:677]: Hashtag in address should be removed, as GitHub renders #number as a pull request link. Same problem exists in `Zaynax Limited`.

**Review by arafat-hasan — CHANGES_REQUESTED** [review]
Addresses must be fixed

File: README.adoc (hunk 2 above)
Comment (mmahmoodictbd) [reply README.adoc]: Fixed

File: README.adoc (hunk 3 above)
Comment (mmahmoodictbd) [reply README.adoc]: Fixed

File: README.adoc (hunk 5 above)
Comment (mmahmoodictbd) [reply README.adoc]: Fixed

File: README.adoc (hunk 6 above)
Comment (mmahmoodictbd) [reply README.adoc]: Fixed

File: README.adoc (hunk 4 above)
Comment (mmahmoodictbd) [reply README.adoc]: Fixed

**Review by arafat-hasan — APPROVED** [review]
I think, we can merge now if you are finished.

**Comment by mmahmoodictbd** [comment]
> I think, we can merge now if you are finished.

Thanks!

**Pull request merged**